tests/Doctrine/Tests/ORM/Functional/ValueObjectsTest.php

```diff
@@ -242,6 +243,48 @@ public function testThrowsExceptionOnInfiniteEmbeddableNesting($embeddableClassN
         ));
     }
 
+    public function testCRUDOfNullableEmbedded()
```

Comment:
I'll also change the name of the classes that I've created on that file to match the issue on JIRA (DDC-3529)